sound/soc/intel/boards/Kconfig

```diff
@@ -674,6 +674,9 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	select SND_SOC_RT715_SDW
 	select SND_SOC_RT715_SDCA_SDW
 	select SND_SOC_RT5682_SDW
+	select SND_SOC_CS42L42_CORE
+	select SND_SOC_CS42L42_SDW
+	select SND_SOC_MAX98363
```

Comment:
Please split cs42l42 and max98363 into two commits

Reply:
Two separate commits are introduced in this PR for max98363 and cs42l42